answer_rocket/config.py

```diff
@@ -216,6 +216,95 @@ def delete_copilot_skill_chat_question(self, copilot_skill_chat_question_id: UUI
         except Exception as e:
             return None
 
+    def create_copilot_question(self, nl='') -> MaxMutationResponse:
```

Comment:
@gbarchard you may want to return a `CreateCopilotionQuestionResponse` object that would inherit from `MaxMutationResponse`, and add the newly added `copilot_question_id`.  That would allow the consumer of the function to turn around and update or delete it using the ID.